Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
+import ImageThumbnail from './ImageThumbnail'
+import HeaderTitle from './HeaderTitle'
+import HeaderContent from './HeaderContent'
 import './Header.css';
+
+const HeaderContainer = props => (
+    <div className='header-container'>
+        <ImageThumbnail thumbnail={props.header.thumbnail}/>
+        <div className='header-text'>
+            <HeaderTitle title={props.header.title}/>
```

Comment:
no need to add the props.header.title.. this can simply be props.header. You can define it as something.header.title at the next class component.